webpack.config.dev.js

```diff
@@ -5,7 +5,7 @@ exports.entry.unshift('webpack/hot/dev-server');
 exports.module.loaders[0].loaders = ['babel', 'react-hot', 'jsx?harmony'];
 exports.plugins[1] = new webpack.DefinePlugin({
   __PROD__: false,
-  __S3_BUCKET__: 'landline'
+  __S3_BUCKET__: JSON.stringify('landline')
```

Comment:
Hmm... I suppose not. That also simplifies the setup function -- you just have to tell Landline its team name and the element to render into. I dig it.